ivy_tests/test_ivy/test_functional/test_core/test_linalg.py

```diff
@@ -1066,7 +1089,10 @@ def test_svd(
     if results is None:
         return
 
-    ret_np_flat, ret_from_np_flat = results
+    ret, ret_from_np = results
+    # flattened array returns
+    ret_np_flat = helpers.flatten(ret=ret)
```

Comment:
this is not needed because of `return_flat_np_arrays=True`

Reply:
Okay, will verify this again, as it seems like something that was added after I made my changes because earlier the function wasn't returning flattened arrays. Thanks!